fmom/pxpypze.go

```diff
@@ -211,3 +211,17 @@ func (p4 *PxPyPzE) Set(p P4) {
 	p4[2] = p.Pz()
 	p4[3] = p.E()
 }
+
+func dotProduct(p1, p2 *PxPyPzE)float64{
+	dotproduct := p1.Px()*p2.Px()+p1.Py()*p2.Py()+p1.Pz()*p2.Pz()
+	return dotproduct
+}
+
+//return the cosine of the angle between the momentum of two jets
```

Comment:
It returns the cosine of an angle and not the angle itself, so it's just a value between -1 and 1. Would it be useful to have a function that returns the angle as well?